src/gazebo_ros_link_attacher.cpp

```diff
@@ -114,6 +117,13 @@ namespace gazebo
 
     ROS_DEBUG_STREAM("Links are: "  << l1->GetName() << " and " << l2->GetName());
 
+    if (teleport_child_to_parent) {
+      auto pose = j.m1->WorldPose();
+      pose.Pos() =
+      j.m1->WorldPose().Pos() + offset;
+      j.m2->SetWorldPose(pose);
+    }
```

Comment:
```suggestion
    if (teleport_child_to_parent) {
      auto pose = j.m1->WorldPose();
      pose.Pos() += offset;
      j.m2->SetWorldPose(pose);
    }
```
F: indentation is off
S: don't need to repeat `j.m1->WorldPose()`